offboard_control.md - use offboard override (#3372)
PX4/PX4-user_guide: en/ros/offboard_control.md

There are two things you want to setup on the firmware side before starting offboard development.

### Enable RC Override

In _QGroundControl_ you can set the [COM_RC_OVERRIDE](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) parameter to automatically switch from offboard mode (or any mode) to Position mode if the RC sticks are moved.

> This is the best way to ensure that an operator can easily take control of the vehicle, and do so in an easy-to-fly mode.

Author: ```suggestion
This is the best way to ensure that an operator can easily take control of the vehicle and switch to the safest flight mode.
```
